code/modules/vehicles/wheelchair.dm

```diff
@@ -165,3 +190,25 @@
 /obj/item/wheelchair/Destroy()
 	wheelchair = null
 	. = ..()
+
+/obj/item/wheelchair/golden
+	name = "gilded wheelchair"
+	icon_state = "wheelchair_folded_gold"
+	custom_materials = list(/datum/material/gold = 20000)
+	var/obj/vehicle/ridden/wheelchair/golden/wheelchair2
+
+/obj/vehicle/ridden/wheelchair/golden/MouseDrop(over_object, src_location, over_location) //cant just use the original because the chair needs to fold into the right version
+	if(over_object != usr || !Adjacent(usr))
+		return FALSE
+	if(!ishuman(usr) || !usr.canUseTopic(src, BE_CLOSE))
+		return FALSE
+	if(has_buckled_mobs())
+		to_chat(usr, span_warning("You need to unbuckle the passenger from [src] first!"))
+		return FALSE
+	usr.visible_message(span_notice("[usr] collapses [src]."), span_notice("You collapse [src]."))
+	var/obj/item/wheelchair/wheelchair_folded = new /obj/item/wheelchair/golden(get_turf(src))
+	forceMove(wheelchair_folded)
+	wheelchair_folded.desc = "A collapsed [name] that can be carried around." 
+	wheelchair_folded.name = name
+	wheelchair_folded.wheelchair = src
+	usr.put_in_hands(wheelchair_folded)
```

Comment:
change object destruction and wrench act to drop gold and stuff

Reply:
yeah I probably should, though the idea of permanently erasing gold is amusing